### PR TITLE
[EASI-2003] GitHub actions

### PR DESCRIPTION
**Pull request by ClayBenson94**
# EASI-2003

## Changes and Description

- GitHub actions

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Review by azeez-salu — APPROVED** [review]

**Review by rileyanderson — COMMENTED** [review]
Once ECR supports caching manifests we should add caching to these image builds. It is currently being [worked on](https://github.com/aws/containers-roadmap/issues/876).

**Review by StevenWadeOddball — DISMISSED** [review]
Thanks Clay, 

I'm providing approval based on the working email configuration and testing them out in DEV.  This looks good! I'd like someone who is more familiar with GitHub to review the workflows  as well.

File: .github/workflows/deploy_to_environment.yml [hunk 1]
```diff
@@ -64,45 +59,53 @@ jobs:
...
         run: |
-          ./scripts/deploy_service "easi-app-db-migrate" "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-db-migrate" "$ECR_REGISTRY/$ECR_REPOSITORY:$GIT_HASH"
-          ./scripts/db_lambda_invoke "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-app-db-migrate"
+          export TASK_REVISION=$(./scripts/update_ecs_task_definition.sh)
+          ./scripts/run_ecs_task.sh
+      # - name: Backup the database
+      #   if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
+      #   run: |
+      #     ./scripts/rds-snapshot-app-db
       - name: Deploy ECS service
```
Comment (rileyanderson) [inline .github/workflows/deploy_to_environment.yml:94]: Maybe in a future iteration we could look into using this [amazon-ecs-deploy-task-definition](https://github.com/aws-actions/amazon-ecs-deploy-task-definition) action

File: .github/workflows/deploy_to_environment.yml [hunk 2]
```diff
@@ -64,45 +59,53 @@ jobs:
...
+          SECURITY_GROUP_NAME: "ecs-db-migrate-${{ inputs.env }}"
         run: |
-          ./scripts/deploy_service "easi-app-db-migrate" "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-db-migrate" "$ECR_REGISTRY/$ECR_REPOSITORY:$GIT_HASH"
-          ./scripts/db_lambda_invoke "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-app-db-migrate"
+          export TASK_REVISION=$(./scripts/update_ecs_task_definition.sh)
+          ./scripts/run_ecs_task.sh
+      # - name: Backup the database
+      #   if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
+      #   run: |
+      #     ./scripts/rds-snapshot-app-db
```
Comment (rileyanderson) [inline .github/workflows/deploy_to_environment.yml:93]: Do we need this?

**Review by rileyanderson — APPROVED** [review]
Looks good! Just a couple small comments but nothing holding it back

**Pull request merged**